<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UploadFile</class>
 <widget class="QWidget" name="UploadFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>587</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTreeWidget" name="tWSelectHomeWork">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>331</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <attribute name="headerVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QTreeWidget" name="tWSelectClass">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>301</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <attribute name="headerVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Class</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gBSelectOtherFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>430</y>
     <width>331</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="lEFileName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>20</width>
     <height>541</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="gBSelectStudent">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>301</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Select Specific Student</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>33</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Student</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cBSelectStudent">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>All</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QPushButton" name="pBUpload">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>530</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBClose">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>540</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rBSelectHomeWork">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select HomeWork Or Note</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rBSelectOtherFile">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>400</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Other File</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pBClose</sender>
   <signal>clicked()</signal>
   <receiver>UploadFile</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>690</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel">
     <x>846</x>
     <y>551</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
